README.md

```diff
@@ -1 +1,9 @@
-# docker-mongodb-pharmacy
\ No newline at end of file
+# docker-mongodb-pharmacy
+
+This is a repo for our docker MongoDB images + data (there are 2 sample files in the `data` directory )
```

Comment:
Maybe delete the unused data file (pharmacy-data-latest.json) to avoid confusion and modify this comment.

When I'm testing stuff I tend to create my own cut down data file.